card/spec/models/card/trash_spec.rb

```diff
@@ -239,18 +239,18 @@
   end
 
   it "test_recreate_plus_card_name_variant" do
-    Card.create( :name => "rta+rtb" ).delete
-    Card["rta"].update_attributes :name=> "rta!"
-    c = Card.create! :name=>"rta!+rtb"
+    Card.create( name: "rta+rtb" ).delete
```

Comment:
Space inside parentheses detected.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.